File: stack/python/Stack.py
class Stack:
    def __init__(self):
        self.stack = []
        self.size = 0

    def push(self, element):
        self.stack.append(element)
        self.size += 1

    def pop(self):
        if not self.empty():
            self.stack.pop()
            self.size -= 1

    def top(self):
        if not self.empty():
            return self.stack[-1]
        return "Empty stack!!!"

    def empty(self):
        if self.size == 0:
            return True
        return False

    def length(self):
        return self.size

File: stack/js/stack.js
class Stack {
    constructor() {
        this.stack = [];
        this.stackLength = 0;
    }

    push(element) {
        this.stack.push(element);
        this.stackLength++;
    }

    pop() {
        if (!this.isEmpty()) {
            this.stack.pop();
            this.stackLength--;
        }
    }

    top() {
        if (!this.isEmpty()) {
            return this.stack[this.length() - 1];
        }
        return "Empty Stack";
    }

    isEmpty() {
        return this.stackLength === 0;
    }

    length() {
        return this.stackLength;
    }
}

const books = new Stack();

books.push("Clean Code");
books.push("Clean Architecture");
books.push("Design Patterns");
books.push("Refactoring");

console.log(books.top());
books.pop();
console.log(books.top());
books.pop();
console.log(books.top());
console.log(books.length());